### ISSUE TITLE
:bug: GitCommandError when doing nothing on the DDA Quant Module page

### ISSUE THREAD

**Issue by enryH** [Proteobench/ProteoBench#259]
**Describe the bug**
Without any interaction an error occurs. 

**To Reproduce**
1. Go to DDA Quant Ion Level model
2. Wait and have an instable internet connection
3. Scroll down and 

**Expected behavior**
App should sit and wait

**Screenshots**

```
GitCommandError: Cmd('git') failed due to: exit code(128) cmdline: git clone -v -- https://github.com/Proteobench/Results_quant_ion_DDA.git 
C:\Users\User\AppData\Local\Temp\tmpe4hc3c55
stderr: 'Cloning into 'C:\Users\User\AppData\Local\Temp\tmpe4hc3c55'... 
POST git-upload-pack (185 bytes) 
POST git-upload-pack (277 bytes) 
error: RPC failed; curl 92 HTTP/2 stream 0 was not closed cleanly: CANCEL (err 8) 
error: 4651 bytes of body are still expected fetch-pack: unexpected disconnect while reading sideband packet fatal: early EOF 
fatal: fetch-pack: invalid index-pack output '
```

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser : Chrome
 - Version latest

I guess we need to catch an error somwhere.

**Comment by enryH** [comment]
```python
File "C:\Users\user\Documents\repos\ProteoBench\webinterface\pages\DDA_Quant_ion.py", line 319, in _run_proteobench
    result_performance, all_datapoints, input_df = IonModule().benchmarking(
File "C:\Users\user\Documents\repos\ProteoBench\proteobench\modules\dda_quant_ion\module.py", line 102, in benchmarking
    all_datapoints = self.add_current_data_point(all_datapoints, current_datapoint)
File "C:\Users\user\Documents\repos\ProteoBench\proteobench\modules\dda_quant_base\module.py", line 285, in add_current_data_point
    all_datapoints["old_new"] = "old"
File "C:\Users\user\Documents\repos\ProteoBench\proteobench\github\gh.py", line 22, in read_results_json_repo
    clone_repo_anon(t_dir, remote_git_repo)
File "C:\Users\user\Documents\repos\ProteoBench\proteobench\github\gh.py", line 13, in clone_repo_anon
    repo = Repo.clone_from(remote_git, clone_dir)
File "C:\Users\user\Anaconda3\envs\proteobench\lib\site-packages\git\repo\base.py", line 1308, in clone_from
    return cls._clone(
File "C:\Users\user\Anaconda3\envs\proteobench\lib\site-packages\git\repo\base.py", line 1219, in _clone
    finalize_process(proc, stderr=stderr)
File "C:\Users\user\Anaconda3\envs\proteobench\lib\site-packages\git\util.py", line 419, in finalize_process
    proc.wait(**kwargs)
File "C:\Users\user\Anaconda3\envs\proteobench\lib\site-packages\git\cmd.py", line 604, in wait
    raise GitCommandError(remove_password_if_present(self.args), status, errstr)
```

This is to get the results json ? Is there a way to get the file locally from a raw github link?

**Comment by RobbinBouwmeester** [comment]
Catch the error here:

proteobench/github/gh.py

**Comment by RobbinBouwmeester** [comment]
Fixed #323
